packages/sdk-server/src/actions.ts

```diff
@@ -192,11 +197,10 @@ export async function verifyOtp(
       );
     }
     const session: Session = {
-      sessionType: "rw",
+      sessionType: SessionType.READ_WRITE,
       userId: userId,
       organizationId: request.suborgID,
-      expiry:
-        Math.floor(Date.now() / 1000) + (request.sessionLengthSeconds ?? 900), //TODO change this to the actual expiry time from the response,
+      expiry: Date.now() + Number(900) * 1000, // TODO: change this to the actual expiry time from the response,
```

Comment:
should we take this on, or part of v2?

```
// TODO: change this to the actual expiry time from the response
```

also, we can remove if no longer relevant. might be useful to specify that this is expressed in MS though (implied by the ` * 1000` and `Date.now()`, but wouldn't hurt)